docs/modules/ROOT/pages/subscriptions/filtering.adoc

```diff
@@ -144,3 +149,32 @@ movieDeleted(where: {genre_NOT: "Comedy"}) {
 This way, only deleted movies of all genres except for `"Comedy"` will trigger events to this subscription.
 
 NOTE: `Where` will only filter by existing properties right before deletion.
+
+=== Combining operators
+
+All above-mentioned operators can be combined using the `AND`/`OR` operators. 
+They accept an array argument with items of the same format as the `where` argument, which means they can also be nested to form complex combinations.
+
+Say we are picky fans of comedy movies and we only accept ratings below 7 for movies released before the 2000's. 
+As an exception we also like the movie "The Matrix". However, we do not like any of its sequels. 
+We could subscribe to any updates that we are interested in as follows:
+
+[source, graphql, indent=0]
+----
+movieUpdate(where: {
+    OR: [
+        {title_ENDS_WITH: "The Matrix"},
+        {AND: [
+            {genre: "comedy"},
+            {OR: [
+                {releasedIn_LTE: 2000},
+                {releasedIn_GT: 2000, averageRating_GT: 7}
+            ]}
+        ]}
+    ]
+}) {
+    updatedMovie {
+        title
+    }
+}
+----
```

Comment:
We could consider just using a link here to the top level filtering page and mention that AND/OR filters are supported